Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+        "IsMuted",
+        _("Video is muted"),
+        _("Return if video is muted"),
+        _("Volume"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isMuted");
+
+    object
+      .addCondition(
+        "Play",
+        _("Is played"),
+        _("Test if an video is played"),
+        _("_PARAM0_  is played"),
+        "Control time",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPlayed");
+
+    object
+      .addCondition(
+        "Pause",
+        _("Is paused"),
+        _("Test if an video is paused"),
+        _("_PARAM0_  is paused"),
+        "Control time",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPaused");
+
+    object
+      .addExpression(
+        "IsPaused",
+        _("Video is played"),
+        _("Return if video is played"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPaused");
+
+    object
+      .addExpression(
+        "IsPlayed",
+        _("Video is played"),
+        _("Return if video is played"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPlayed");
+
+    object
+      .addCondition(
+        "Loop",
+        _("Is looped"),
+        _("Is looped"),
+        _("_PARAM0_  is looped"),
+        "Control time",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isLooped");
+
+    object
+      .addExpression(
+        "IsPaused",
+        _("Video is paused"),
+        _("Return if video is paused"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPaused");
+
+    object
+      .addExpression(
+        "IsLooped",
+        _("Video is looped"),
+        _("Return if video is looped"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isLooped");
+
+    object
+      .addCondition(
+        "Volume",
+        _("Volume"),
+        _("Test the volume of an video object"),
+        _("Volume of _PARAM0_ is _PARAM1_ _PARAM2_"),
```

Comment:
No extra spaces, this is inconsistent with other extensions: _("Volume of _PARAM0_ is _PARAM1__PARAM2_")